doc_src/tutorial.rst

```diff
@@ -750,6 +750,7 @@ Now in another shell::
     > echo $EDITOR
     vim
 
+See more on universal variables :ref:`here <universal-variables>` (especially on how to set them once).
```

Comment:
This sounds weird to me - setting them once is easy, you just `set -U`.

The important part is that you only *need* to set them once.

How about:

```
You only need to set universal variables once, interactively. There's no need to add them to your :ref:`config files <initialization>`.

For more, see :ref:`Universal Variables <universal-variables>`
```

?

Reply:
Done! (With minor edits.) WDYT?